contracts/DummyDNSSEC.sol

```diff
@@ -0,0 +1,27 @@
+pragma solidity ^0.4.17;
+
+contract DummyDNSSEC {
+    uint16 expectedType;
+    bytes expectedName;
+    uint32 inception;
+    uint64 inserted;
+    bytes20 hash;
+
+    function setData(uint16 _expectedType, bytes _expectedName, uint32 _inception, uint64 _inserted, bytes _proof) public {
+      expectedType = _expectedType;
+      expectedName = _expectedName;
+      inception = _inception;
+      inserted = _inserted;
+      if(_proof.length == 0) {
+          hash = bytes20(0);
```

Comment:
don't really need this part. Do the inversed, don't set if length == 0

Reply:
Changed to the followings

```
      if(_proof.length != 0) {
          hash = bytes20(keccak256(_proof));
      }
```